lab-allie/app/service/auth-service.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+module.exports = [
+  '$q',
+  '$log',
+  '$http',
+  '$window',
+  // 'authService',
+  function($q, $log, $http, $window, authService) {
```

Comment:
I think this was my own typo in the lecture, but don't need that `authService` argument.